lambda-backend/src/components/mongo/index.js

```diff
@@ -0,0 +1,45 @@
+const debug = require('debug')('mood:mongo');
+const { MongoClient } = require('mongodb');
+
+module.exports = config => {
+	const start = async () => {
+		const mongo = await MongoClient.connect(config.url, config.options);
+		const db = mongo.db(config.db);
+		debug('Configuring db....');
+		db.collection('points').createIndex({ id: 1 }, { unique: true });
+		db.collection('points').createIndex({ level1: 1, level2: 1, level3: 1 }, { sparse: true });
+		db.collection('points').createIndex({ team: 1, timestamp: 1 });
+
+		const getMoodByTeam = async team => {
+			debug(`Getting latest mood points from mongo for team ${team}...`);
+			const points = await db.collection('points').find({ team }).toArray();
```

Comment:
From a frontend perspective (point drawing) - It could be a good idea to set a limit in order to retrieve just the points within e.g. the last two weeks. This way we could end up sending to the frontend e.g. 1000points and currently the frontend would draw them with a 0.1opacity each. The frontend could also do this filtering step but I think it´s obviously way better to do it in the backend

Reply:
You're absolutely right. However documents in S3 are being kept for a week, so you wouldn't really retrieve too many of them for now. It's not included in the purpose of this PR but we will definitely build that later on once we need a more refined search.

Reply:
👍🏻